bootq/run-debug.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+export GENNY_SHOW_VALUES="TRUE"
+export GENNY_KAFKA_URL=alyson.genny.life:9092
+export GENNY_REALM=${GENNY_REALM:internmatch}
```

Comment:
this needs to be GENNY_KEYCLOAK_REALM. Will default to gada in app properties otherwise